resources/client_install.rb

```diff
@@ -18,6 +18,8 @@
 provides :mariadb_client_install
 unified_mode true
 
+include MariaDBCookbook::Helpers
+
```

Comment:
This is a better way of doing this.
```suggestion
action_class do
  include MariaDBCookbook::Helpers
end
```

Reply:
Done in bb4f1724baad27d4dc531f082f44144782b9c14d